modules/source/mail/config/mail.xml

```diff
@@ -34,7 +38,7 @@
 			store-uri="${protocol}://${username:}:${password:}@${host}:${port}/${folder}"
```

Comment:
> We should assume that users won't have access to edit the file and also different stream instances may want to use different properties. I guess the location isn't as important - if we want to reduce duplication for shared config.

I agree. We can add these properties into module XML and drive them based on profile.
If someone wants to add any more JavaMail properties, then can we still assume the module XML is the place to add them?